jwst/datamodels/cube.py

```diff
@@ -8,40 +8,37 @@ class CubeModel(DataModel):
     """
     A data model for 3D image cubes.
 
-    Parameters
-    ----------
-    init : any
-        Any of the initializers supported by `~jwst.datamodels.DataModel`.
+    Attributes
+    __________
+    data : numpy float32 array
+         The science data
 
-    data : numpy array
-        The science data.  3-D.
+    dq : numpy uint32 array
+         Data quality array
```

Comment:
All of the new text strings have lost the "2-D", "3-D" type of comments from the originals. Is there a way to keep these somewhere (or some indication of the dimensionality)?

Reply:
I'll open a separate issue for this.